lms/static/js/dashboard/legacy.js

```diff
@@ -79,6 +79,35 @@
              return properties;
          }
 
+         function setDialogAttributes(isPaidCourse, certNameLong,
+                                        courseNumber, courseName, enrollmentMode, showRefundOption) {
+             var diagAttr = {};
+
+             if (isPaidCourse) {
+                 if (showRefundOption) {
+                     diagAttr['data-refund-info'] = 'You will be refunded the amount you paid.';
```

Comment:
Yes. We should use gettext so we can get these strings translated. We do need to show Spanish version of our site now.